docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -63,6 +56,20 @@ $ sudo usermod -a -G libvirtd <username>
 $ newgrp libvirtd
 ----
 
+. As root, install the KVM driver binary and make it executable as follows (For Ubuntu-16.04 and higher):
++
+----
+# curl -L https://github.com/dhiltgen/docker-machine-kvm/releases/download/v0.10.0/docker-machine-driver-kvm-ubuntu16.04 -o /usr/local/bin/docker-machine-driver-kvm
+# chmod +x /usr/local/bin/docker-machine-driver-kvm
+----
+
+. For Ubuntu-14.04:
++
+----
+# curl -L https://github.com/dhiltgen/docker-machine-kvm/releases/download/v0.10.0/docker-machine-driver-kvm-ubuntu14.04 -o /usr/local/bin/docker-machine-driver-kvm
+# chmod +x /usr/local/bin/docker-machine-driver-kvm
+----
```

Comment:
@praveenkumar This section needs some update. 

1. First I think we can remove `Debian` from the section name i.e. only `On Ubuntu` is fine instead of `On Debian/Ubuntu`.  Because we do not have binary built for Debian and we do not know which binary  (there are two binary or Ubuntu now) will work on which version of Debian. Also, I have not seen any Debian user in Minishift. In case there are any, they can figure that out from Ubuntu versions.
2. The installation text for Ubuntu 16.04 and 14.04 needs to be consistent.  Check the sentences before the `curl` command. 
`As root, install the KVM driver binary and make it executable as follows (For Ubuntu-16.04 and higher):` 
vs 
`For Ubuntu-14.04:`

Reply:
I don't think this is necessary to change, it clearly explains what needs to be done. It first gives the general steps for installing and then goes on to distinguish between the two version based on the binary needed.

```
. As root, install the KVM driver binary and make it executable as follows: 

- For Ubuntu-16.04 and higher:
......................................
- For Ubuntu-14.04:
..................................
```

Reply:
@anjannath Why would you want o mention Debian if there are no binary built specifically for Debian and we do no test for Debian? What about consistency in language?